src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx

```diff
@@ -0,0 +1,638 @@
+* Do not hyphenate New Relic or its platform, capabilities, and so on, and do not allow them to break across a page line when used in text. 
+* Avoid confusion by clearly separating and identifying New Relic trademarks from other companies' names and/or trademarks. 
+These trademark guidelines apply to other companies' trademarked names and products too.
+
+## Naming
+See when to use title case (first word and principal words are capitalized), what not to capitalize, and what to use and not use on first and subsequent uses.
+Note that first use is referencing the first mention in the body copy. It's okay to use the subsequent versions in space-constrained areas such as titles, headers, tables, web navigation, the UI, social media posts, and so on.
+
+## When to use title case
+
+You must use title case for our trademarked names including our company name plus our product/platform name and a few capability and integration names. 
+
+<table>
+    <thead>
+        <tr>
+            <th style={{ width: "150px" }}>
+                Name</th>
+            <th>What it is
+            </th>
+            <th>Use this
+            </th>
+            <th>Not this
+            </th>
+        </tr>
+    </thead>
+    
+    <tbody>
+        
+        <tr>
+            <td>New Relic*</td>
+            <td>our company</td>
+            <td>First use: New Relic, Inc. (corporation/entity), New Relic® (printed assets), or New Relic (digital assets)
+
+            Subsequent use: New Relic, our company, we, or it
+            </td>
+            <td>New Relic's, new relic, New relic, NR, their
+            </td>
+        </tr>
+
+        <tr>
+            <td>New Relic One</td>
+            <td>our product/ our platform</td>
+            <td>First use: New Relic One observability platform
+
+                Subsequent uses: New Relic One or New Relic One platform
+            </td>
+            <td>New Relic One's, New Relic one, NR1
+            </td>
+        </tr>
+
+        <tr>
+            <td>FutureStack
+
+                \{Future\}Stack
+            </td>
+            <td>our annual user group conference
+            </td>
+            <td>First use: FutureStack® or \{Future\}Stack® (printed assets), or FutureStack or \{Future\}Stack (digital assets)
+                
+                Subsequent uses: FutureStack
+            </td>
+            <td>Future Stack, Futurestack, Future stack
+            </td>
+        </tr>
+
+        <tr>
+            <td>NerdGraph*</td>
+            <td>our GraphQL API</td>
+            <td>First use: NerdGraph® (printed assets) or NerdGraph (digital assets)
+                
+                Subsequent uses: NerdGraph</td>
+            <td>Nerd Graph, Nerdgraph, nerdgraph, nerd graph
+            </td>
+        </tr>
+
+        <tr>
+            <td>Nerdlet*</td>
+            <td>component that make up New Relic One apps; a specific UI view represented by a React JavaScript package</td>
```

Comment:
few of these have 'component that make up...', maybe should be 'a component of New Relic One apps'...

Reply:
fixed!